cfg.yaml

```diff
@@ -12,17 +12,18 @@ routes:
 - name: route1                                 #  Route name. Must be unique
   input: contains(input.image, "alpine")       #  REGO rule to match input message against route
   outputs: [my-slack]                          #  Output name (needs to be defined under "outputs") which will receive the message
-  template: slack-template                     #  Template name (needs to be defined under "templates") which will be used to process the message output format
+  template: vuls-slack                         #  Template name (needs to be defined under "templates") which will be used to process the message output format
   plugins:                                     #  Optional plugins
    aggregate-issues-number:                       # Number of same messages to aggregate into one output message
    aggregate-issues-timeout:                      # Number of seconds/minutes/hours to aggregat same messages into one output. Maximum is 24 hours. Use Xs or Xm or Xh
    policy-show-all: true                          # Send a message to output even if exact same message was already processed. Default is false.
+   unique-message-props: ["digest","image","registry"] # Optional: Comma separated list of top level properties which unique identify an event message. If message with same id is received more than once it will be ignored
```

Comment:
Should it be "input.digest", "input.image" and "input.registry"?

Reply:
Checking top level properties was the simplest implementation. I see 2 options to extend it:
# use rego (this way `input.*` is required)
# use own logic to traverse input (this will allow to use path like `user.email`)